examples/common.h

```diff
@@ -77,6 +77,12 @@ std::string gpt_random_prompt(std::mt19937 & rng);
 
 std::vector<llama_token> llama_tokenize(struct llama_context * ctx, const std::string & text, bool add_bos);
 
+//
+// Model utils
+//
+
+struct llama_context * load_model_from_file(gpt_params & params);
```

Comment:
```suggestion
struct llama_context * llama_init_from_gpt_params(const gpt_params & params);
```

Reply:
Function name changed, and branch rebased/pushed.

Reply:
The argument should be `const` reference

Reply:
Oops! Fixed.